src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseEventTrigger.java

```diff
@@ -23,6 +23,8 @@
     private Boolean showEventNotes;
     @JsonProperty("end_button_label")
     private String endButtonLabel;
+    @JsonProperty("can_save_draft")
```

Comment:
why not more simply `draftEnabled`?

Reply:
This is consistent with excel file hence the naming.